cmd/mountDockerSock.go

```diff
@@ -0,0 +1,64 @@
+package cmd
+
+import (
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+// DockerSockPath is the mount path of the Docker socket
+const DockerSockPath = "/var/run/docker.sock"
+
+func checkMountDockerSock(container ContainerV1, result *Result) {
+	if container.VolumeMounts != nil {
+		for _, mount := range container.VolumeMounts {
+			if mount.MountPath == DockerSockPath {		
+				occ := Occurrence{
+					container: container.Name,
+					id:        ErrorDockerSockMounted,
+					kind:      Warn,
+					message:   "/var/run/docker.sock is being mounted, please avoid this practice.",
```

Comment:
I would change this to " ..., please avoid mounting docker socket on your containers"